client/src/banners/active-banner.tsx

```diff
@@ -203,7 +203,8 @@ export default function ActiveBanner() {
   }
 
   const isEnabled = (id: string) =>
-    userData.waffle.flags[id] && !isEmbargoed(id);
+    (userData.waffle.flags[id] || userData.waffle.switches[id]) &&
```

Comment:
I think you should tag your PR description to also "Fixes #1252"
https://github.com/mdn/yari/issues/1252